File: telegram-bot-bumblebee-core/src/main/java/com/github/bumblebee/command/youtube/dao/YoutubePostedVideosRepository.java
package com.github.bumblebee.command.youtube.dao;

import com.github.bumblebee.command.youtube.entity.PostedVideo;
import org.springframework.data.repository.CrudRepository;
import org.springframework.stereotype.Repository;

/**
 * Created by Fare on 25.11.2016.
 */
@Repository
public interface YoutubePostedVideosRepository extends CrudRepository<PostedVideo, String> {
}

File: telegram-bot-bumblebee-core/src/main/java/com/github/bumblebee/command/youtube/dao/YoutubePostedVideosRepository.kt
package com.github.bumblebee.command.youtube.dao

import com.github.bumblebee.command.youtube.entity.PostedVideo
import org.springframework.data.repository.CrudRepository
import org.springframework.stereotype.Repository

@Repository
interface YoutubePostedVideosRepository : CrudRepository<PostedVideo, String>
